src/extension.ts

```diff
@@ -31,14 +31,29 @@ export function activate(context: ExtensionContext) {
       commands.registerCommand(PICK_FROM_RECENT_COMPARES, pickFromRecents),
       commands.registerCommand(CLEAR_RECENT_COMPARES, globalState.clear),
   );
-
-  if (process.env.COMPARE_FOLDERS === 'DIFF') {
+  var conf = workspace.getConfiguration("compareFolders");
+  var folderLeft = conf.get('folderLeft','').trim();
+  var folderRight = conf.get('folderRight','').trim();
+  if (!folderLeft || ! folderRight)
```

Comment:
That means that everyone who don't use this feature will see the this message. Maybe it will be better to skip it if the user doesn't want to use the feature.
Maybe if one of them is not empty but the second one does, we can notify the user that they need to set it up.

Reply:
And I think this is now solve too